frontend/src/components/Employees/ViewPay.tsx

```diff
@@ -0,0 +1,187 @@
+import * as React from 'react';
+import { DataGrid, GridColDef, GridSelectionModel } from '@material-ui/data-grid';
+import { useEffect, useState } from 'react';
+import { Button, Grid, Typography } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+import { getAllBusinessPays } from '../../services/LogHoursAPI';
+import { PayStatus } from '../../dto/LogHours/PayDTOs';
+import EditIcon from '@material-ui/icons/Edit';
+import { useAppSelector } from '../../redux/hooks';
+import { selectAccount } from '../../features/account/AccountSlice';
+
+interface DataDisplay {
+  id: number;
+  employee: string;
+  hoursWorked: number;
+  hourlyWage: number;
+  paidAmount: number;
+  paidStatus: PayStatus;
+  startDate: string;
+  endDate: string;
+}
+
+const ViewPay: React.FC = () => {
+  const [paysList, setPaysList] = useState<DataDisplay[]>([]);
+  const [selectedPays, setSelectedPays] = useState<DataDisplay[]>([]);
+  const [totalHours, setTotalHours] = useState<number>(0);
+  const [totalPay, setTotalPay] = useState<number>(0);
+
+  const history = useHistory();
+
+  const clickAddPay = () => {
+    history.push(`/pay/new`);
+  };
+
+  const clickEditPay = (id: number) => {
+    history.push(`/pay/edit/${id}`);
+  };
+
+  const updateSelection = (selectionModel: GridSelectionModel) => {
+    const selected = paysList.filter((pay) => selectionModel.indexOf(pay.id) > -1);
+    setSelectedPays(selected);
+  };
+
+  useEffect(() => {
+    var hoursSum = 0;
```

Comment:
Maybe use `let` instead of var. It is usually a standard i think in ES6.

Reply:
Good catch, I'll change it